### DISCUSSION TITLE
I need help on git rebase

### DISCUSSION THREAD

**Discussion by Markokondicdev** [community/community#31428] [Other Feature Feedback, Questions, & Ideas] [answered]
There are two branches test1 and test2. The goal of this challenge is to use git rebase to bring both commits onto master. When finished there should be no merge commits or branching

**Comment by airtower-luna** [comment, marked as answer]
That sounds a bit more like homework and less like a question. See [Git Branching - Rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) to figure out an answer. It'll involve rebasing and fast-forward merging the branches one after the other. :wink:
